### DISCUSSION TITLE
Next templates - suggestions and voting

### DISCUSSION THREAD

**Discussion by jankapunkt** [jankapunkt/latexcv#63] [Ideas]
Please submit your ideas and upvote. If you have an interesting template for your idea as input, please add one or two links.

**Comment by jankapunkt** [comment]
## Scientific Template

... was mentioned before so I add it as first idea. 

However, I found it hard to find **appealing** scientific templates as inspiration. Many of them were either boring or not covering the requirements for applying in scientific areas. How can we make a scientific template compliant **and** visually appealing?

**Comment by skilljobs** [comment]
Hey can you make some similar templates which are available to novoresume?
